exercises/acronym/acronym_test.go

```diff
@@ -0,0 +1,47 @@
+package acronym
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input     string
+	expected  string
+	expectErr bool
+}
+
+var stringTestCases = []testCase{
+	{"Portable Network Graphics", "PNG", false},
+	{"HyperText Markup Language", "HTML", false},
+	{"Ruby on Rails", "ROR", false},
+	{"PHP: Hypertext Preprocessor", "PHP", false},
+	{"First In, First Out", "FIFO", false},
+	{"Complementary metal-oxide semiconductor", "CMOS", false},
```

Comment:
I would suggest that we remove the `expectsErr` from `testCase`, remove the `, false` from all cases, and make `abbreviate` return just a `string` rather than `(string, error)`. What say you?

What I am thinking is - does the act of forming an acronym perform any operation that could possibly fail? We can look to [`strings`](https://golang.org/pkg/strings/) as guidance - it has no error on [ToUpper](https://golang.org/pkg/strings/#ToUpper) (or indeed any of its functions except those on `Reader` and `Replacer`)